NodeAFK.js

```diff
@@ -0,0 +1,215 @@
+const desktopIdle = require('desktop-idle');
+const EventEmitter = require('events');
+
+const STATUS_IDLE = 'idle';
+const STATUS_ACTIVE = 'active';
+
+/**
+ * NodeAFK
+ *
+ * @extends EventEmitter
+ */
+class NodeAFK extends EventEmitter {
+  /**
+   * Create a new instance of NodeAFK
+   * @param {number} inactivityDuration The duration until the user is considered `idle` (in ms)
+   * @param {number} pollInterval How often should NodeAFK poll the system to
+   * get the current idle time (in ms)
+   * @param {string} initialStatus The initial status of the user
+   */
+  constructor(
+    inactivityDuration,
+    pollInterval = 1000,
+    initialStatus = STATUS_ACTIVE,
```

Comment:
I'm not sure does it actually make sense to allow setting the inital status? If the modules been invoked, the user has to be interacting with the computer, which means they will always be classed as active initally

Reply:
I was trying to think use cases outside of how we are using this module. It could be that somebody else using this module wants the initial status to be idle for some reason?

I suppose they could just call `setStatus` though. I'll remove the `initialStatus` in the constructor for now in favour of using `setStatus` 👍